FinSearchUnified/Bundle/StoreFrontBundle/Gateway/Findologic/CustomFacetGateway.php

```diff
@@ -77,7 +79,7 @@ public function getFacetsOfCategories(array $categoryIds, ShopContextInterface $
         $criteria->addCondition(new CategoryCondition($categoryIds));
 
         $query = $this->queryBuilderFactory->createProductQuery($criteria, $context);
-        $response = $query->execute();
+        $response = $query->execute()->getRawResponse();
```

Comment:
Split this up in two variables, as in the next story we will use the FINDOLOGIC-API responses. One should contain the raw response, the other one the response object.